pkg/server/server.go

```diff
@@ -167,6 +178,24 @@ func WithExperimentals(experimentals ...ExperimentalFeatureFlag) OpenFGAServiceV
 	}
 }
 
+func WithCheckQueryCacheEnabled(enabled bool) OpenFGAServiceV1Option {
```

Comment:
Can we add godoc to these, like we did elsewhere? This is part of our public API (if used as a lib), so it feels pertinent.